docs-v2/spec.yaml

```diff
@@ -437,6 +437,21 @@ paths:
                                                 metadata:
                                                     type: object
                                                     description: Custom metadata attached to the connection
+                                                errors:
+                                                    type: array
+                                                    items:
+                                                        type: object
+                                                        properties:
+                                                            type:
+                                                                type: string
+                                                                example: "'auth' or 'sync'"
+                                                            log_id:
+                                                                type: string
+                                                                example: "VrnbtykXJFckCm3HP93t"
+                                                    description: |
+                                                        List of connection errors. Ex:
+                                                        - Connection credentials are invalid (type=auth)
```

Comment:
is this description actually correct since we only give back a log id?

Reply:
I didn't document the log_id field since it can only be used with our UI.
This PR is a small step in the direction of exposing connection level errors so customers know something is wrong for a given connection.  Eventually we might want to add a `reason` but for now it is technically complicated to get the details of one single log from ES